private/pkg/protosource/option_extension_descriptor.go

```diff
@@ -126,3 +126,16 @@ func isDescendantPath(descendant, ancestor []int32) bool {
 	}
 	return true
 }
+
+func (o *optionExtensionDescriptor) PresentFields() map[protoreflect.FieldDescriptor]protoreflect.Value {
+	// Note: This does not bother to handle unrecognized extensions.
```

Comment:
Since this returns all fields, not just extensions:
```suggestion
	// Note: This does not bother to handle unrecognized fields.
```